core/src/modules/configLoaders/app.ts

```diff
@@ -74,15 +74,17 @@ export async function loadApp(
     if (!appRefreshQuery) {
       appRefreshQuery = await AppRefreshQuery.create({
         appId: configObject.id,
-        refreshQuery: configObject.refreshQuery,
+        refreshQuery: configObject.refreshQuery.query,
+        recurringFrequency: configObject.refreshQuery.recurringFrequency,
```

Comment:
is it possible that `configObject.refreshQuery` is undefined?  This might be safer:

```suggestion
        refreshQuery: configObject.?refreshQuery.query,
        recurringFrequency: configObject.?refreshQuery.recurringFrequency,
```

Reply:
Same comment in the `else` case below

Reply:
I think it should actually check that `configObject.refreshQuery?.length > 1` at the top of the entire if block (line 65)... otherwise if someone leaves the defaults in the config file it would create an empty appRefreshQuery for them, which doesn't make sense.  

Adding both.